framework/grid/DataColumn.php

```diff
@@ -177,6 +177,9 @@ protected function renderFilterCellContent()
 
         $model = $this->grid->filterModel;
 
+        // The default options the filterCell has
+        $defaultFilterOptions = ['class' => 'form-control', 'id' => null];
+
```

Comment:
* this $variable can be moved closer to the please where it is used
* it could also directly be merged with $filterInputOptions
* the prompt can be merged below